content/tutorial/tutorial.md

```diff
@@ -100,35 +100,35 @@ cd ..
 
 4. `index.css` adlı faylı `src/` direktoriyasına [bu CSS kodu](https://codepen.io/gaearon/pen/oWWQNa?editors=0100) ilə əlavə edin.
 
-5. `index.js` adlı faylı `src/` directoriyasına [bu JS kodu](https://codepen.io/gaearon/pen/oWWQNa?editors=0010) ilə əlavə edin.
+5. `index.js` adlı faylı `src/` direktoriyasına [bu JS kodu](https://codepen.io/gaearon/pen/oWWQNa?editors=0010) ilə əlavə edin.
 
-6. `src/` directoriyasında olan `index.js` faylının ən yuxarısınz aşağıdalı üç sətri əlavə edin:
+6. `src/` direktoriyasında olan `index.js` faylının başlanğıcına aşağıdakı üç sətri əlavə edin:
 
 ```js
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 ```
 
-İndi, əgər siz layihə direktoriyasından `npm start` icra etsəniz və brauzerdə `http://localhost:3000` səhifəni açsanız, boş "X O oyunu" taxtası.
+İndi, əgər siz layihə direktoriyasından `npm start` icra etsəniz və brauzerdə `http://localhost:3000` səhifəsini açsanız, boş "X O oyunu" lövhəsini görməlisiniz.
 
-Editorunuz üçün sintaks seçilməsini konfiqurasiya etmək üçün [bu təlimatları](https://babeljs.io/docs/editors/) izləməniz tövsiyyə olunur.
+Redaktorunuzda sintaktik işıqlandırılmanı sazlamaq üçün [bu təlimatlara](https://babeljs.io/docs/editors/) əməl etməyiniz tövsiyə olunur.
 
 </details>
 
-### Kömək, Mən İlişmişəm! {#help-im-stuck}
+### Kömək edin, Mən İlişmişəm! {#help-im-stuck}
 
-Əgər ilişmisinizsə, [cəmiyyətimizin dəstək resurslarını](/community/support.html) nəzərdən keçirin. Xüsusilə, [Reactiflux Chat](https://discord.gg/reactiflux) kömək almaq üçün ən tez yoldur. Əgər siz cavab almırsınızsa və ya siz yenədə ilişmisinizsə, xahiş olunur bizə İssue göndərin və biz sizə kömək etməyə cəhd edəcəyik.
+Əgər ilişmisinizsə, [cəmiyyətimizin dəstək resurslarını](/community/support.html) nəzərdən keçirin. Tez bir zamanda kömək almaq üçün [Reactiflux Chat](https://discord.gg/reactiflux) xüsusilə əla bir yoldur. Əgər cavab almırsınızsa və ya yenə də ilişmisinizsə, xahiş olunur bizə Github üzərindən Issue göndərin və biz sizə kömək etməyə cəhd edəcəyik.
 
 ## İcmal {#overview}
 
-İndi sizdə hər şey qurulub deyə gəlin of React haqqında məlumat alaq!
+Quraşdırmanı artıq tamamladığınıza görə, gəlin indi React haqqında məlumat alaq!
 
 ### React Nədir? {#what-is-react}
 
-React UI yaratmaq üçen deklarativ, səmərəli, və elastik Javascript kitabxanasıdır. Bu kitabxana sizə "komponent" adlanan balaca və sərbəst kod hissələrinin birləşməsi ilə kompleks UI yaratmağa icazə verir.
+React - istifadəçi interfeysləri (UI) yaratmaq üçün deklarativ, səmərəli və elastik Javascript kitabxanasıdır. Bu kitabxana sizə "komponent" adlanan kiçik sərbəst kod hissələrindən mürəkkəb UI yaratmağa icazə verir.
```

Comment:
Why is there dash ("-") here? Maybe, comma (",")?

Reply:
From a grammatical point of view it seems more appropriate. In any case, it does not matter so much. We can use both of them.